### DISCUSSION TITLE
Compatibility with prisma3.13.0

### DISCUSSION THREAD

**Discussion by IvanDandrea86** [MichalLytek/typegraphql-prisma#282] [Q&A]
Any planned update to make compatible with the latest version of prisma 3.13.0?

**Comment by MichalLytek** [comment]
Of course, as always. Point me when I haven't updated the generator to work with new version. You can always disable the version check if no breaking changes introduced.

**Reply by IvanDandrea86** [reply]
Thanks for the quick answare, I have disabled version control and it works perfect!

**Reply by sxnney** [reply]
What do you mean by disabling the version control? I'm unable to get `typegraphql-prisma` to work with `prisma 3.13.0`, it gives me `error - Error: Looks like you've forgot to provide experimental metadata API polyfill. Please read the installation instruction for more details.` even though I've configured everything as it is in the installation guide.
